src/components/entries/EntriesContainer.js

```diff
@@ -54,9 +53,6 @@ export default createClass({
       contentElement = <Placeholder content='Please select your Entry.' />
     }
 
-    if (entries.payload.len() < entries.total) {
-      loadMoreElement = <FeaturelessButton label='Load more' action={this.loadEntries} />
-    }
```

Comment:
The load more functionallity was not fully implemented. This was also the only implementation of the `FeaturelessButton`. So I removed the code and the button itself.